sprimber-cucumber/src/main/java/com/griddynamics/qa/sprimber/discovery/CucumberSuiteDiscovery.java

```diff
@@ -101,6 +103,17 @@ private void testCaseNodeDiscover(Node parentNode, CucumberDocument cucumberDocu
                 });
     }
 
+    private void fillSuiteHooks(Node suiteNode) {
+        cucumberTestBinder.fillPreConditions("Before Suite", suiteNode);
+        cucumberTestBinder.fillPostConditions("After Suite", suiteNode);
+    }
+
+    private void fillFeatureHooks(Node suiteNode) {
```

Comment:
1. Please rename the parameter, looks like it's not suiteNode, it's testCaseNode
2. Why do you call fillFeatureHooks for testCase? What is the relation between testCase and Feature?

Reply:
1 - done
2 - it is the same